src/main/java/com/google/api/codegen/viewmodel/testing/ApiTestCaseView.java

```diff
@@ -14,6 +14,16 @@
  */
 package com.google.api.codegen.viewmodel.testing;
 
+import com.google.api.codegen.viewmodel.InitCodeView;
+
+import java.util.List;
+
 public class ApiTestCaseView {
-  // TODO(shinfan): Implement this.
+  public String name;
+  public String methodName;
+  public String requestType;
+  public String resourceType;
```

Comment:
put "Name" on the end of the fields holding type names: "requestTypeName", "resourceTypeName"